### ISSUE TITLE
fix bahaviour on long press

### ISSUE THREAD

**Issue by mokurin000** [ritiek/piano-rs#48]
imo, it's strange that piano-rs will detect long press as rapid clicks

**Comment by ritiek** [comment]
I don't think crossterm (and more so the version used in piano-rs) allows to detect key releases. I'm not sure if it's even possible to detect key up presses in terminals. Might want to use a GUI to properly implement things like this.

**Comment by mokurin000** [comment]
> I don't think crossterm (and more so the version used in piano-rs) allows to detect key releases. I'm not sure if it's even possible to detect key up presses in terminals. Might want to use a GUI to properly implement things like this.

yeah... may need switch to a gui backend (and render as what a terminal emulator do), thus need few of work
